apps/console/src/extensions/application-templates/shared/components/m2m-custom-configuration.tsx

```diff
@@ -0,0 +1,384 @@
+                                                tab.
+                                            </Trans>
+                                        </Hint>
+                                    </label>
+                                    <CopyInputField
+                                        value={ copyScopesValue }
+                                        data-componentid={ `${ componentId }-scope-readonly-input` }
+                                    />
+                                </Form.Field>
+                                <Form.Field>
+                                    <label>
+                                        {
+                                              t("extensions:console.application.quickStart" +
+                                              ".m2m.customConfig.tokenRequest")
+                                        }
+                                        <Hint className="mt-0 mb-0" popup>
+                                            {
+                                                t("extensions:develop.applications.quickstart" +
+                                                ".m2m.configurations.tokenRequest")
+                                            }
+                                        </Hint> 
+                                    </label>
+                                    <CodeEditor
+                                        oneLiner
+                                        readOnly="nocursor"
+                                        withClipboardCopy
+                                        showLineNumbers={ false }
+                                        language="shell"
+                                        options={ {
+                                            lineWrapping: true
+                                        } }
+                                        height="100%"
+                                        theme="dark"
+                                        sourceCode={
+                                            tokenRequest
+                                        }
+                                    />
+                                </Form.Field>
+                            </Form>
+                        </Grid.Column>
+                    </Grid.Row>
+                </Grid>
+            </>
+        );
+    };
+
+    return (
+        <Card
+            fluid
+            className="basic-card no-hover quick-start-custom-config-message no-background"
+        >
+            <Card.Content textAlign="left">
+                <Text muted>
+                    <Trans
+                        i18nKey={
+                            "extensions:develop.applications.quickstart" +
+                            ".m2mApp.configurations.heading"
+                        }
+                    >
+                        A Machine-to-Machine Application represents a program that interacts with an API without any user involvement. 
+                        Execute a{" "}
```

Comment:
'Execute a..' looks odd IMO. Have we got the text content reviewed by the doc team?

Reply:
Adding @divyaamunugama